conan/tools/cmake/cmakedeps/templates/config.py

```diff
@@ -28,6 +28,23 @@ def additional_variables_prefixes(self):
             self.cmakedeps.get_property("cmake_additional_variables_prefixes", self.conanfile, check_type=list) or [])
         return list(set([self.file_name] + prefix_list))
 
+    @property
+    def parsed_extra_variables(self):
+        # Reading configuration from "cmake_extra_variables" property
+        from conan.tools.cmake.utils import parse_extra_variable
+        conf_extra_variables = self.conanfile._conanfile.conf.get("tools.cmake.cmaketoolchain:extra_variables",
```

Comment:
This is the main reason I didn't go with this route from the start. Should the conf attribute be exposed to the `ConanFileInterface` class? Or should this private access remain?

Reply:
Good point. Lets give access via ``ConanFileInterface``, it is already possible to read ``settings``, ``options``, ``info``, why not ``conf`` too?